projects/smoldot-extension/package.json

```diff
@@ -9,11 +9,13 @@
     "last 2 versions"
   ],
   "staticFiles": {
-    "staticPath": "src/assets"
+    "staticPath": "src/assets",
+    "watcherGlob": "**"
   },
   "scripts": {
-    "build": "parcel build src/popup.html src/background.ts src/content.ts --no-source-maps --public-url ./",
-    "dev": "parcel watch src/popup.html src/background.ts src/content.ts --public-url ./",
+    "clean": "rm -rf dist",
+    "build": "yarn run clean && parcel build src/*.html src/*.ts --no-source-maps --public-url ./",
```

Comment:
Is this right?  `parcel` expects a list of *entry points* and creates one bundle for each entry point